docs/blog/09-25-2024-introducing-the-federation-api.md

```diff
@@ -0,0 +1,404 @@
+  without authentication or authorization.
+- **Registered access**, which means that authentication (but not authorization)
+  is required to access the files (e.g., you have to sign up for an account, but
+  you don't need any special permissions beyond that).
+- **Controlled access** data, which means that data access must be explicitly
+  granted through some data request process.
+
+Each site has different file sharing procedures and generally will have
+different requirements for authentication and authorization.
+
+To capture and describe this information, the API specification introduces the
+concept of a `Gateway`: a novel mechanism for defining the authentication and
+authorization requirements for accessing a particular set of data. The available
+gateway kinds today are `Open`, `Registered`, `Controlled`, and `Closed` (made
+available for files that are not _yet_ available, but will be made available at
+some point in the future).
+
+`Gateway`s wrap `Link`s, which describe how the data can be accessed once you've
+fulfilled the authentication and authorization requirements for that data. There
+are a variety of different avenues for data access supported by the
+specification, including:
+
+- **`Direct` links**, which describe an exact URL that may be accessed. This is
+  useful is the most straightforward case where the data is available by
+  visiting the provided URL.
+- **`Approximate` links**, which provide a URL and then subsequent instructions
+  to follow to access the data. This is useful for when (a) an exact link does
+  not exist (e.g., "go to this page and use the filters on the right") and (b)
+  when describing how to access data that cannot be made available through a
+  single link (e.g., requesting all of the data for a particular sample).
+- **`Informational` links**, which point to an external webpage where you can
+  learn more about how to access the data. This is useful when the source server
+  itself does not have direct knowledge of how to access the data, but it can
+  direct you to a URL where you can learn more.
+- **`MailTo` links**, which support listing an email address that you can
+  contact about accessing the data.
+
+That being said, this list may not be all inclusive today, and growing the list
+of supported link types is an area of future discussion.
+
+Together, the `Gateway` concept (covering _how_ to access the data) combined
+with the `Link` concept (covering _where_ to access the data) provides an
+expressive vocabulary for articulating how data may be accessed within the
+ecosystem. Our goal is to improve the user experience of accessing data shared
+by various federation members and to make it easier to build automated data
+access pipelines that seamlessly download the latest pediatric cancer research
+data as it becomes available. Several examples of how to assign and interpret
+these `Gateway`s/`Link`s are available [in the corresponding documentation]
+(**Clay to add link here**).
```

Comment:
You are probably aware of this, but just wanted to point out that this placeholder still exists in the text.